src/GrpcProtos/capture.proto

```diff
@@ -677,6 +677,12 @@ message ErrorEnablingUserSpaceInstrumentationEvent {
   bytes message = 2;
 }
 
+message InfoEnablingUserSpaceInstrumentationEvent {
+  uint64 timestamp_ns = 1;
+  // This is a string, we use bytes to avoid UTF-8 validation.
+  bytes message = 2;
```

Comment:
I am wondering whether we want this generic event with the string, or a specific "Can not instrument function event" with a function ID instead.

Reply:
`InstrumentationResult` contains the association between function ID and error message. We should definitely keep it all the way to the client. For example we might want to show functions instrumented with uprobes differently, ask to not instrument them, etc.